modules/check_stop_condition.py

```diff
@@ -0,0 +1,57 @@
+"""
+Checks whether the drone has reached it's max flight time and sends it back to launch.
+"""
+
+import dronekit
+
+from . import upload_commands
+
+
+MAVLINK_LANDING_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL
+MAVLINK_LANDING_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_RETURN_TO_LAUNCH
```

Comment:
These are not landing commands, rename the variables.

Reply:
I matched the variable names from the `add_takeoff_and_landing_command.py` module. What should I change them to?

Reply:
`MAVLINK_RTL_...`